Missfits Code 2018 PowerUp/src/org/usfirst/frc/team6418/robot/Robot.java

```diff
@@ -169,4 +339,46 @@ public void teleopPeriodic() {
 	@Override
 	public void testPeriodic() {
 	}
+	
+	public int MAKINGTHISAREDXcheckZone(boolean groundSwitchPressed, boolean switchSwitchPressed, boolean scaleSwitchPressed, 
```

Comment:
which switch is which?

Reply:
`switch (switchSwitch) { ...`

Reply:
its the switch switch! the limit switch for the "SWITCH" level of the elevator :)